code/game/gamemodes/events/holidays/Holidays.dm

```diff
@@ -1,26 +1,15 @@
-//Uncommenting ALLOW_HOLIDAYS in config.txt will enable Holidays
-var/global/Holiday = null
-
-//Just thinking ahead! Here's the foundations to a more robust Holiday event system.
-//It's easy as hell to add stuff. Just set Holiday to something using the switch (or something else)
```

Comment:
Why did you remove these instructions?

Reply:
I guess I'll add my own instructions, I simplified the code so it just returns the holiday using GetHoliday() rather than assigning it directly to the Holiday global